packages/shared_preferences/shared_preferences_tool/pubspec.yaml

```diff
@@ -0,0 +1,24 @@
+name: shared_preferences_tool
+description: "DevTools extension for package:shared_preferences. Manage SharedPreferences efficiently. Edit, search, and view keys."
+publish_to: 'none'
+
+version: 1.0.0+1
+
+environment:
+  sdk: '>=3.4.0 <4.0.0'
+
+dependencies:
+  devtools_app_shared: ^0.1.1
+  devtools_extensions: ^0.1.1
```

Comment:
please use the latest available package versions:
- `devtools_app_shared: ^0.2.3` (https://pub.dev/packages/devtools_app_shared/changelog)
- `devtools_extensions: ^0.2.2` (https://pub.dev/packages/devtools_extensions/changelog)

Reply:
I see in the changelog that we now have `caseInsensitiveFuzzyMatch` and `DevToolsClearableTextField`. I'll use them instead of the custom code that I've implemented.